frog/server/mergeData.js

```diff
@@ -12,7 +12,7 @@ export default (activityId: string, object: ObjectT) => {
   const { activityData } = object;
   const activity = Activities.findOne(activityId);
 
-  const [groups, activityStructure] = getInstances(activityId);
+  const { groups, structure } = getInstances(activityId);
```

Comment:
Very nice that we can change the calling code so little, and get so much more clarity! :)